examples/kafka-hub/A Guide on implementing Websub Hub backed by Kafka Message Broker.md

```diff
@@ -25,7 +25,7 @@ Following are some of the goals and technical requirements which we considered w
 # Implementation 
 As mentioned above implementation is based on Kafka message broker. Therefore, the Kafka message broker will be doing most of the heavy lifting. At a high level following are the key components associated with this implementation.
 
-![image](kafka_hub_image.png)
+![image](../https://github.com/ballerina-platform/module-ballerina-websubhub/tree/main/examples/kafka-hub/_resources/images/kafka_hub_image.png)
```

Comment:
Isn't this the correct way?

```suggestion
![image](https://github.com/ballerina-platform/module-ballerina-websubhub/tree/main/examples/kafka-hub/_resources/images/kafka_hub_image.png)
```

Reply:
Fixed with [eb2fff6ab439d5aaeecceead59150cd597eba8f7](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/918/commits/eb2fff6ab439d5aaeecceead59150cd597eba8f7)